tests/dopamine/discrete_domains/run_async_training_test.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for run_experiment_test.py."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from dopamine.discrete_domains import run_experiment
+from dopamine.utils import test_utils
+from tensorflow import test
+
+
+class AsyncRunnerTest(test.TestCase):
+  """Tests for asynchronous trainer."""
+
+  def testLocalEnvironment(self):
+    """Tests that environment is managed locally."""
```

Comment:
"managed locally" is hard to interpret (and maybe consists of more than just initialization).  I'd favor using language and naming the test something that's as close to the thing being tested as possible (here, initialization is a proxy for management).

Reply:
I'm not, in this case, totally able myself to suggest a different name.  I know the tested behavior has something to do with initialization, but I don't remember enough about how `runner.run_experiment` works to anticipate what's being tested.

Reply:
Done. Updated name and docstring.